kafka-zookeeper/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM alpine:3.12 AS kafka
+
+ENV SCALA_VERSION 2.12
+ENV KAFKA_VERSION 2.3.0
+
+WORKDIR /kafka
+COPY install.sh /kafka/install
+RUN /kafka/install && rm /kafka/install
+
+COPY wait-for-zookeeper.sh /kafka/bin
+COPY start.sh /kafka/bin
+
+# Share the same base image to reduce layers
+FROM hypertrace/java:11
+
+MAINTAINER Hypertrace "https://www.hypertrace.org/"
+
+WORKDIR /kafka
+
+RUN adduser -g '' -h /kafka -D kafka
+
+COPY --from=kafka --chown=kafka /kafka /kafka
+
+USER kafka
```

Comment:
Declare this above and use $USER in the create command and chown commands.

Reply:
you want to set `USER` environment variable?

Reply:
done.